.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Do we want to add `package-ecosystem: npm`? Since there is a `package.json` here, this seems like something to add.

Reply:
Wise.  Added in commit 4837792.